samples/msal-node-samples/client-credentials/README.md

```diff
@@ -43,11 +43,17 @@ const config = {
     auth: {
         clientId: "ENTER_CLIENT_ID",
         authority: "https://login.microsoftonline.com/ENTER_TENANT_INFO",
-        clientSecret: "ENTER_CLIENT_SECRET",
+        process.env.CLIENT_SECRET,
     },
 };
 ```
 
+You will also need to add the client secret to your **.env file**:
```

Comment:
What is the env file? Is it not just an env variable?

Reply:
In all of our node samples, we use the dotenv package to load everything defined in the .env file into environment variables.